Src/main.c

```diff
@@ -106,15 +112,25 @@ int main(void)
 
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
+
+  const uint8_t TIMER_LOW = 1;
+  const uint8_t TIMER_HIGH = 2;
+
   while (1)
   {
-  /* USER CODE END WHILE */
 
+  /* USER CODE END WHILE */
+    
   /* USER CODE BEGIN 3 */
+    uint16_t adcValue = HAL_ADC_GetState(&hadc);
```

Comment:
this isn't the right function to call. You'l have to look through some more hal docs to find the correct one.